tests/certification/pubsub/azure/servicebus/topics/servicebus_test.go

```diff
@@ -1355,6 +1355,7 @@ func TestServicebusWithSessionsRoundRobin(t *testing.T) {
 
 func componentRuntimeOptions() []runtime.Option {
 	log := logger.NewLogger("dapr.components")
+	log.SetOutputLevel(logger.DebugLevel)
```

Comment:
Do you need this to be merged?

Reply:
I think it's helpful to debug failures, some logs are only debug level. But I can revert if you prefer

Reply:
Please, revert this to avoid regression to log levels.

Reply:
Oh. It is just cert tests. LGTM